### DISCUSSION TITLE
Why has development slowed down?

### DISCUSSION THREAD

**Discussion by patoi** [carbon-design-system/carbon-components-svelte#1568] [General]
https://github.com/carbon-design-system/carbon-components-svelte/graphs/commit-activity

There have been no commits on master since October 20th.

**Comment by theetrain** [comment]
We're currently coming up with a roadmap for carbon-components-svelte v1, which includes Carbon v11 feature parity. Stay tuned for announcements!

**Comment by theetrain** [comment]
As promised, here's our roadmap announcement! https://github.com/carbon-design-system/carbon-components-svelte/discussions/1614

**Comment by metonym** [comment]
Development slowed down in 2022 because the author left IBM.

However, in the past six months, active development has resumed. Over 200 issues have been completed: https://github.com/carbon-design-system/carbon-components-svelte/issues?q=is%3Aissue%20is%3Aclosed%20closed%3A%3E%3D2025-09-01
